### PR TITLE
Snap

### PR DESCRIPTION
**Pull request by willmcgugan**
### What this PR solves
- Adds file logging
- Tweaks snap name

### Review
- [X] Ready for review

### CODE REVIEW

**Comment by willmcgugan** [comment]
If I were to merge this without a review, would that be an abuse of my admin powers? Slack outage is a massive pain.

File: dataplicity/app.py [hunk 1]
```diff
@@ -3,12 +3,13 @@
 
 import argparse
 import logging
+import logging.config
 import sys
 
 from . import __version__
 from . import subcommand
+from .subcommands import *
```
Comment (beardygeek) [inline dataplicity/app.py]: Is there a need for the wildcard import?

Comment (willmcgugan) [reply dataplicity/app.py]: No, you're right. Better to be explicit.

**Comment by willmcgugan** [comment]
@barfle Doesn't agent already log to `/var/log/dataplicity.log` ?

**Comment by barfle** [comment]
It does, but I think it's via syslog no?

**Comment by willmcgugan** [comment]
Actually no, it was supervisor redirecting stdout to /var/log/syslog.

**Comment by barfle** [comment]
Yes but it was still using syslog right?  The reason being is in the IoT space, people can have some pretty random ways of managing logs, and if they are routed via syslog they can control all of them in one place rather than every app having its own config or way of doing things....

**Comment by willmcgugan** [comment]
No, we've never used syslog with agent, AFAIK

I can make it use syslog, if you want to.

**Comment by barfle** [comment]
Right now it's a case of maintaining the status quo.  If we don't use syslog presently, either via our code or indirectly via supervisor, and we are presently writing out to the file directly, then that's fine just carry on and we'll address it later.  If we are indirectly using supervisor to collect dataplicity logs through syslog, then we should maintain writing via syslog.

**Pull request merged**